src/klines.ts

```diff
@@ -39,7 +39,7 @@ export async function getKline(
       data = [...data, ...response.data];
     }
   }
-
+  --data.length;
```

Comment:
I'm not sure, why did you add this ?

Reply:
The last candle is unnecessary because it will be the beginning of a new candle. If you load, for example, 5-minute candles 27-07-2022 and 28-07-2022 and connect them together, you will get two identical candles in the place where they will be connected. Simply put, the last candle 27-07-2022 = the first candle 28-07-2022, you get a repeat candle (double). Check it out for yourself.